artmaker/src/commonMain/kotlin/io/fbada006/artmaker/utils/ExportHelper.kt

```diff
@@ -17,4 +17,9 @@ package io.fbada006.artmaker.utils
 
 import androidx.compose.ui.graphics.ImageBitmap
 
-expect suspend fun shareImage(imageBitmap: ImageBitmap?)
+/**
+ * Called to trigger sharing an image depending on the platform
+ *
+ * @param imageBitmap is the image to share
+ */
```

Comment:
So as you can see @emmanuelmuturia, the approach I have taken is only providing docs to the expect calls. We can think of them as interfaces. As such, we only need to document the base method and not the implementations.